### ISSUE TITLE
Improve the content of the tooltip of a project

### ISSUE THREAD

**Issue by mathbagu** [powsybl/powsybl-gse#35]
Currently, the tooltip of a project shows its description
```
Project: <description>
```
If a project has no description, the content is not very interesting:
```
Project:
```
The tooltip should show the complete path (if not too long), and the description of the project:
```
Project: <path of the project>
Description: <description>
```
The description line should be shown only if the description is set.

**Comment by MioRtia** [comment]
Partly resolved by #111: Description only appears if it is set
TO BE ADDED: the tool tip should display the path

**Comment by MioRtia** [comment]
Fix #123
